index.bs

```diff
@@ -11347,9 +11347,10 @@ must be [=%IteratorPrototype%=].
     1.  Let |kind| be |object|’s kind.
     1.  Let |values| be the list of [=value pairs to iterate over=].
     1.  Let |len| be the length of |values|.
-    1.  If |object|’s index is greater than or equal to |len|, then
+    1.  If |index| is greater than or equal to |len|, then
         return [=CreateIterResultObject=](<emu-val>undefined</emu-val>, <emu-val>true</emu-val>).
     1.  Let |pair| be the entry in |values| at index |index|.
+    1.  Set |index| to |index| + 1.
```

Comment:
This should instead set _object_'s index to _index_ + 1 so that the state is stored.